src/olympia/conf/prod/settings.py

```diff
@@ -198,6 +198,17 @@
 if NEWRELIC_ENABLE:
     NEWRELIC_INI = '/etc/newrelic.d/%s.ini' % DOMAIN
 
+FXA_CONFIG = {
+    'client_id': env('FXA_CLIENT_ID'),
+    'client_secret': env('FXA_CLIENT_SECRET'),
+    'content_host': 'https://accounts.firefox.com',
+    'oauth_host': 'https://oauth.accounts.firefox.com/v1',
+    'profile_host': 'https://profile.accounts.firefox.com/v1/profile',
```

Comment:
Actually I'm unsure of this URL but we can try it on stage and see if it works. It need might to be `https://profile.accounts.firefox.com/v1`.